lib/discordrb/data/reaction.rb

```diff
@@ -16,12 +22,30 @@ class Reaction
     # @return [String] the name or unicode representation of the emoji
     attr_reader :name
 
+    # @return [true, false] whether the current bot or user used this reaction as a burst reaction
+    attr_reader :me_burst
+    alias_method :me_burst?, :me_burst
```

Comment:
i was going to nitpick this but then i realized this is what discord calls it, dumb. in the future we can workshop a less caveman sounding alias